bigquery/google/cloud/bigquery/table.py

```diff
@@ -1722,9 +1735,9 @@ class TimePartitioning(object):
             Number of milliseconds for which to keep the storage for a
             partition.
         require_partition_filter (bool, optional):
-            If set to true, queries over the partitioned table require a
-            partition filter that can be used for partition elimination to be
-            specified.
+            DEPRECATED: Use
```

Comment:
Are there any annotations we can/should add to help docs/and existing consumers to understand this is no longer the preferred mechanism?

Reply:
👍 Added deprecation warning message in 84e2c3cca903d78e56706c5e777ff4d9b322ef79